frontends/mrxl/test/runt.toml

```diff
@@ -0,0 +1,38 @@
+ver = "0.4.0"
+
+[[tests]]
+name = "interpret"
+paths = [ "./*.mrxl" ]
+cmd = "mrxl {} --data {}.data --interpret"
+expect_dir = "expect/interpret_out/"
+
+[[tests]]
+name = "gen_futil"
+paths = [ "./*.mrxl" ]
+cmd = "mrxl {}"
+expect_dir = "expect/futil_out/"
+
+[[tests]]
+name = "convert_data"
+paths = [ "./*.mrxl" ]
+cmd = "mrxl {} --data {}.data --convert"
+expect_dir = "expect/data_out/"
```

Comment:
It's not _100%_ clear to me that we need separate tests for data conversion, since any problems there will manifest also when we try to do any of the `simulate_*` tests… but I also don't really object since it is cheap to do and, if nothing else, will help explain where stuff went wrong.